src/server/routes.py

```diff
@@ -11,7 +12,8 @@
 def home(lang, year):
     config = get_config(year)
     ebook_size_in_mb = get_ebook_size_in_mb(lang, year)
-    return render_template('%s/%s/index.html' % (lang, year), config=config, ebook_size_in_mb=ebook_size_in_mb)
+    return render_template('%s/%s/index.html' % (lang, year), config=config,
+                           get_file_date_info=get_file_date_info, ebook_size_in_mb=ebook_size_in_mb)
```

Comment:
Why does the home page need to know how many MB the ebook is?

Reply:
Every page now does since we show it in the ToC menu. Moved it to the `render_template` function as per below.

Reply:
Added the ebook file size generation to JSON it's loaded at start up and doesn't need to be calced on the fly any more.